doc/vhpidirect/examples/arrays.rst

```diff
@@ -0,0 +1,63 @@
+.. program:: ghdl
+.. _COSIM:VHPIDIRECT:Examples:arrays:
+
+Arrays
```

Comment:
I've been feeling that Arrays is bit curious after titles like wrapping, linking, shared etc. Could think about updating to "C Cosim: Arrays"?

Reply:
Actually, since wrapping, linking and shared have very few examples, I think that we could move them into the quickstart. Let's wait until we merge a few more PRs to see how we make it all fit. Once we have all the basic features we want, it'll be easier to see the whole picture.